worker-jobtracking-container/src/test/java/com/hpe/caf/worker/jobtracking/JobTrackingWorkerIT.java

```diff
@@ -105,6 +112,67 @@ public void testTrackingReportTasks() throws Exception
         }
     }
 
+    @Test
+    @Description("Report a task progression and update the job" +
+            "Prevent from reporting more than 99% completion")
+    public void testProgressReportingMessages() throws Exception
+    {
+        // Create job
+        final String jobTaskId = jobDatabase.createJobId();
+        jobDatabase.createJobTask(defaultPartitionId, jobTaskId, "");
+
+        // Validate Waiting status
+        DBJob job = jobDatabase.getJob(defaultPartitionId, jobTaskId);
+        assertEquals(JobStatus.Waiting, job.getStatus());
+
+        // Report the final task fully completed
+        final String finalTaskId = jobTaskId+".2*";
+        jobDatabase.reportTaskProgress(defaultPartitionId, finalTaskId, 100);
```

Comment:
These are the worker integration tests but they seem to be bypassing the worker and updating the database directly.  It seems to defeat the purpose.  I would think the new tests should be more like the pre-existing `testTrackingReportTasks()` test (i.e. they send a message to the worker and then only contact the database to verify that the worker has done what is expected).